content/api/alerts.apib

```diff
@@ -1,205 +1,322 @@
     + name (string) - Display name of the alert.
-    + description (string) - Description of the alert.
-    + id (string) - Unique alphanumeric id.
-    + email_addresses (array) - List of email addresses to send alert notifications.
-    + alert_metric (enum) - Metric that the alert is configured to track.
-        + `"monthly_sending_limit"`
-    + threshold (object) - Configuration for level and threshold at which an alert is generated.
-        + error (object) - Error level alert configuration.
-            + target (number) - Percentage value at which an alert will be generated.
-            + comparator (enum) - Conditional comparison operator used by alerts service to compare actual value of `alert_metric` to the `target` value.
-                + `"gt"`
-    + enabled (boolean) - Whether or not the alert is active.
+    + subaccount (number) - Subaccount filter applied to the alert.
+    + any_subaccount (boolean) - Flag indicating all subaccounts should be evaluated for this alert.
+    + metric (enum) - Metric that the alert is configured to track.
+        + `"monthly_sending_limit", "health_score"`
+    + muted (boolean) - Flag indicating whether the alert should be ignored for evaluation/emission.
+    + filters (array) - Filter objects to apply to the alert.
+    + threshold_evaluator (object) - Evaluator object for an alert with a trigger threshold.
+      + operator (enum) - Operator for alert evaluation to use.
+        + `"gt", "lt"`
+      + source (enum) - Source of value to compare against.
+        + `"raw", "day_over_day", "week_over_week"`
+      + value (number) - Target value to compare against.
+    + channels (object) - Channels to send alert notifications through.
+        + emails (array) - Email addresses to send alert notifications to.
 
 + Sample
-        
-    ```
-    {
-        "name": "Sparky's Monthly Alert",
-        "description": "Example alert that will send a notification when 90% of monthly sending limit is used",
-        "id": "cbe0faf0-9051-11e8-ad04-490e82764b23",
-        "email_addresses": [
-            "sparky@sparkpost.com"
-        ],
-        "alert_metric": "monthly_sending_limit",
-        "threshold": {
-            "error": {
-                "target": 90,
-                "comparator": "gt"
-            }
-        },
-        "enabled": true
+
+  ```
+  {
+    "id": 117,
+    "name": "Alert Name Here",
+    "subaccount": 0,
+    "any_subaccount": false,
+    "metric": "health_score",
+    "muted": false,
+    "filters": [
+      {
+        "filter_type": "sending_domain", 
+        "filter_values": ["test.com"]
+      }
+    ],
+    "threshold_evaluator": {
+      "operator": "gt",
+      "source": "raw",
+      "value": 0
+    },
+    "channels": {
+        "emails": [
+            "test@sparkpost.com",
+            "bigwig@test.com"
+        ]
     }
-    ```
+  }
+  ```
 
-### Create an Alert [POST /labs/alerts]
+### Create an Alert [POST /v1/customer-alerts]
```

Comment:
Should be `/v1/alerts` since the old one is still under `/labs/alerts`

Reply:
Yeah, good catch